packages/cli/src/commands/generate/scaffold/templates/components/Name.tsx.template

```diff
@@ -83,15 +94,15 @@ const ${singularPascalName} = ({ ${singularCamelName} }) => {
       </div>
       <nav className="rw-button-group">
         <Link
-          to={routes.${editRouteName}({ id: ${singularCamelName}.id })}
+          to={routes.${editRouteName}({ id: ${singularCamelName}?.id })}
```

Comment:
If this is a text template, it cares that `${singularCamelName}?.id` the template replacement text might be null? Interesting.

Reply:
This isn't to do with the template actually. it just adds the ? after the name.

For example it would be `Post?.id`